<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Timer</class>
 <widget class="QWidget" name="Timer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Timer</string>
  </property>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>561</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QRadioButton" name="slowButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>200</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lent</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="mediumButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>250</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Moyen</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="fastButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>300</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Rapide</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>441</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana</family>
     <pointsize>20</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
